torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -210,7 +210,12 @@ class TORCH_API RpcAgent {
 
   // Stop accepting requests and shutdown the RPC framework as soon as possible
   // by terminating all RPC threads.
-  virtual void shutdown() = 0;
+  virtual void shutdown();
```

Comment:
Sorry, just noticed this - does it make sense to make it non-virtual?  (and if that's the case, to comment on it being on purpose)
I thought the idea is that subclasses would override shutdownImpl(), not this method?

Reply:
It should be non-virtual, I'll update the PR.